sass/cd/cd-header/_cd-user-menu.scss

```diff
@@ -241,3 +226,13 @@
 .region-header-top .menu > li a {
   @extend .cd-user-menu__item;
 }
+
+// Login/Logout item as last item in user menu
+.region-header-top .menu > li:last-child {
+  @extend .cd-user-menu__icon-login;
+}
+
+// Add content menu item as first item in user menu
+.logged-in .region-header-top .menu > li:first-child {
+  @extend .cd-user-menu__icon-plus;
```

Comment:
I added a double-dash to my example `@extend` setup since the plus icon is just a variant of the icon styles.